website/src/error-documentation.mjs

```diff
@@ -84,6 +81,14 @@ markdownParser.renderer.rules = {
   },
 };
 
+function codeHasBOM(codeHTML) {
+  const dom = new jsdom.JSDOM(codeHTML);
+  const mark = dom.window.document.querySelector("mark");
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector, this selected mark will be the first one.